host-configs/rzansel-blueos_3_ppc64le_ib_p9-clang@8.0.1_nvcc_xlf.cmake

```diff
@@ -7,79 +7,80 @@
 # SPDX-License-Identifier: (BSD-3-Clause)
 #------------------------------------------------------------------------------
 # SYS_TYPE: blueos_3_ppc64le_ib_p9
-# Compiler Spec: clang@8.0.0_nvcc_xlf
+# Compiler Spec: clang@8.0.1_nvcc_xlf
 #------------------------------------------------------------------------------
-# CMake executable path: /usr/WS1/axom/thirdparty_libs/builds/2020_01_30_13_08_34/clang-8.0.0_nvcc_xlf/cmake-3.9.6/bin/cmake
+# CMake executable path: /usr/WS1/axom/devtools/blueos_3_ppc64le_ib_p9/latest/cmake-3.9.6/bin/cmake
 #------------------------------------------------------------------------------
 
 #------------------------------------------------------------------------------
 # Compilers
 #------------------------------------------------------------------------------
 
-# C compiler used by spack
-set(CMAKE_C_COMPILER "/usr/tce/packages/clang/clang-8.0.0/bin/clang" CACHE PATH "")
+set(CMAKE_C_COMPILER "/usr/tce/packages/clang/clang-8.0.1/bin/clang" CACHE PATH "")
 
-# C++ compiler used by spack
-set(CMAKE_CXX_COMPILER "/usr/tce/packages/clang/clang-8.0.0/bin/clang++" CACHE PATH "")
+set(CMAKE_CXX_COMPILER "/usr/tce/packages/clang/clang-8.0.1/bin/clang++" CACHE PATH "")
 
-# Fortran compiler used by spack
 set(ENABLE_FORTRAN ON CACHE BOOL "")
 
-set(CMAKE_Fortran_COMPILER "/usr/tce/packages/xl/xl-2019.06.12/bin/xlf2003" CACHE PATH "")
+set(CMAKE_Fortran_COMPILER "/usr/tce/packages/xl/xl-2019.12.23/bin/xlf2003" CACHE PATH "")
 
 #------------------------------------------------------------------------------
 # TPLs
 #------------------------------------------------------------------------------
 
 # Root directory for generated TPLs
-set(TPL_ROOT "/usr/WS1/axom/thirdparty_libs/builds/2020_01_30_13_08_34/clang-8.0.0_nvcc_xlf" CACHE PATH "")
+set(TPL_ROOT "/usr/WS1/axom/libs/blueos_3_ppc64le_ib_p9/2020_02_13_15_16_57/clang-8.0.1_nvcc_xlf" CACHE PATH "")
 
-# conduit from uberenv
 set(CONDUIT_DIR "${TPL_ROOT}/conduit-0.5.1" CACHE PATH "")
 
-# mfem from uberenv
-set(MFEM_DIR "${TPL_ROOT}/mfem-4.0" CACHE PATH "")
+# MFEM not built
```

Comment:
@white238 -- could you please re-enable ``mfem`` within our specs?

#### Details: ####
Due to ``mfem``'s optional dependency on ``sidre``, the ``spack`` version of ``axom/package.py`` should not depend on ``mfem``.  However, for our internal usage, i.e. these ``host-configs``, we want to enable ``mfem`` so we can exercise and test this functionality.

In the near future, I intend to revisit the circular dependency between ``mfem`` and ``axom``.  I was thinking that the mfem ``SidreDataCollection`` needs to be updated, but it would be ideal for this to live in ``axom``, rather than in ``mfem``.  I'll have to work through the logistics of how this might work...

Reply:
After this work is done will we still depend on MFEM or will they depend on us?

Reply:
Once everything is done, I'd like ``axom`` to optionally depend on ``mfem`` and to remove ``mfem``'s dependence on ``axom``.

In this case, ``axom`` will use ``mfem`` for ``primal``/``quest`` functionality related to curved high order elements. ``mfem``'s dependence on ``axom`` is currently limited to a single class (``SidreDataCollection``) which, I think, could live in ``axom``.

Reply:
> Once everything is done, I'd like `axom` to optionally depend on `mfem` and to remove `mfem`'s dependence on `axom`.
> 
> In this case, `axom` will use `mfem` for `primal`/`quest` functionality related to curved high order elements. `mfem`'s dependence on `axom` is currently limited to a single class (`SidreDataCollection`) which, I think, could live in `axom`.

I'd prefer the `SidreDataCollection` to reside with the host code that actually uses that. I don't think there are strong uses cases for making that available elsewhere given that `mfem` already works with `conduit`.

I agree that `mfem` should not be a strong dependency for `axom`.